public/styles.css

```diff
@@ -0,0 +1,36 @@
+body{
+height:100vh;
+}
+.font-bold {
+font-weight: bold;
+}
+ .brush{
+content: "";
+background-image: url(images/brush-stroke.svg);
+background-repeat: no-repeat;
+background-position: -300px 160%;
+background-size: 31rem 34rem;
+position:bottom;
+} 
+
+
+.round{
+border-radius:16px;
+}
+
+.text-dark-blue{
+color:rgb(23, 54, 94);
+}
+
+.text-blue{
+    color:rgb(0, 91, 152);
+    }
+
+.hover-shadow:hover{
+        box-shadow: 0 .125rem .25rem rgba(0,0,0,.075)!important;
+}
+
+.hover-circle:hover{
+    background-color: #e0e1e1!important;
```

Comment:
🗣️ Can this important be removed without any negative effect?

Reply:
probably, i think it was a copy paste error.